src/templates/_macros/app_tile.html

```diff
@@ -3,72 +3,79 @@
 {% from "_macros/previews.html" import preview_tray %}
 {% from "_macros/stars.html" import stars %}
 
-{% macro app_tile(app, feed_app=None, is_detail=False, tray=False, src=None) %}
+{% macro app_tile(product, feed_app=None, is_detail=False, tray=False, src=None) %}
```

Comment:
i think code comments were addressed to not use this for add-ons, but there's still a lot of remnants from the attempt